employee_service/src/main.py

```diff
@@ -72,6 +72,8 @@ def update_data(empid):
         return Response("{'error': 'Erro addding the item'}", mimetype='application/json', status=500)
     return Response(json.dumps(update_emp_res), mimetype='application/json', status=200)
 
+
```

Comment:
the routes names from the above code can be improved.
`@app.route('/addemployee',methods=['POST'])`
Like, `addemployee` can be named as `employee`, the api method will tell what the api will do.

For update. `put` method can be used, as in the below case instead of `post`
`@app.route('/updatedetails/<empid>', methods=['POST'])`

Even the below route names can be improved:
`@app.route('/getemployee/<empid>',methods=['GET'])`
`@app.route('/dept/<string:deptname>', methods = ['GET'])`
`@app.route('/allemployees', methods = ['GET'])`